### PR TITLE
Update README with latest dart pub convention

### PR DESCRIPTION
**Pull request by JaysonBH**
* Change deprecated standalone pub to dart pub
* Remove deprecated Stagehand
* Include dart recommended webdev for web builds

### CODE REVIEW

**Comment by JaysonBH** [comment]
@kevmoo Please Review

File: README.md [hunk 1]
```diff
@@ -21,20 +21,27 @@ local directory.
...
 
 ```console
-$ stagehand web-angular
-$ pub get
-$ pub build
-$ dhttpd --path build/web/  # Serves app at http://localhost:8080
+$ dart create -t web-simple web-app
+$ cd web-app
+$ dart pub get
+$ webdev build
```
Comment (kevmoo) [inline README.md:43]: This should be `dart build` or `dart run build_runner build` or something.

Did you test this?

File: README.md [hunk 2]
```diff
@@ -27,21 +27,15 @@ Otherwise you can use the `dart pub global` command.
...
 Here's an example of creating a web app
 and then running it with this server:
 
 ```console
 $ dart create -t web-simple web-app
 $ cd web-app
 $ dart pub get
-$ webdev build
-$ dhttpd --path build/  # Serves app at http://localhost:8080
+$ dart run build_runner build -o build
```
Comment (kevmoo) [inline README.md:37]: Did you validate this works @JaysonBH ?

File: README.md (hunk 1 above)
Comment (JaysonBH) [reply README.md]: Yes, this was tested. This is what some of the documentation recommended, but I see now that this is for a development flow. Updated with your recommended build_runner.

File: README.md (hunk 2 above)
Comment (JaysonBH) [reply README.md:37]: @kevmoo Yes, I have validated that this works.

Comment (kevmoo) [reply README.md:37]: 👍

**Pull request merged**